pyproject.toml

```diff
@@ -7,15 +7,15 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<3.13"
```

Comment:
The 13 was intended so that 12 is included. Alternatively, <= 3.12

Reply:
FYI llama-index 0.10.13 added support for python 3.12 (see https://github.com/run-llama/llama_index/pull/11377)